pkgs/by-name/si/signal-desktop-bin/signal-desktop-aarch64.nix

```
@@ -1,7 +1,7 @@
 { callPackage }:
 callPackage ./generic.nix { } {
   pname = "signal-desktop-bin";
-  version = "7.52.0";
+  version = "7.54.0-1";
```

```suggestion
  version = "7.54.0";
```
based on the previous version I think we want this instead

---

Indeed, I forgot to remove it this time. I changed the update script so it removes `-1`.